src/ipsis/course-merge/CourseMergeMergedOfferingCollectionEntity.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * SelectedCourseMergeOfferingCollectionEntity class representation of a list of course merge offering as defined in the LMS
```

Comment:
We probably want to change these 2 commented lines to the correct class/serializer